curl/README.md

```diff
@@ -16,7 +17,7 @@ file must be publicly readable, since no credentials are passed in the request.
 
 ```
 steps:
-- name: gcr.io/cloud-builders/curl
+- name: 'curlimages/curl'
   args: ['http://www.example.com/remotefile.zip', '--output', 'localfile.zip']
```

Comment:
This doesn't work because the new curl image runs as a non root user and can't write locally

Reply:
ewww...

I'll update the example to one that works...